src/App.php

```diff
@@ -702,8 +704,12 @@ public function runFrontend(App\Router $router, IManagePersonalConfigValues $pco
 				$module = $router->getModule();
 			}
 
+			// Processes data from GET requests
+			$httpinput = $httpInput->process();
+			$input     = array_merge($httpinput['variables'], $httpinput['files'], $request ?? $_REQUEST);
+
 			// Let the module run it's internal process (init, get, post, ...)
-			$response = $module->run($_POST, $_REQUEST);
+			$response = $module->run($input);
```

Comment:
@annando - Here's the PHP Input stream processing now. It's called **before** any sub-module is called, so it's independent of PUT/DELETE/POST/GET sub-calls and now delivered with every call as `$request` parameter